source/server/controllers/v1/couldDoController.js

```diff
@@ -0,0 +1,67 @@
+import chalk from 'chalk'
+import {
+  newCouldDo,
+  editCouldDo,
+  deleteCouldDo
+} from '../../../dataServices/database/commands/couldDo'
+
+const handleNewCouldDo = ( request, response ) => {
+  const attributes = request.body
+
+  return newCouldDo( attributes )
+    .then( result => {
+      if ( result instanceof Error ) {
+        throw new Error( result )
+      } else {
+        response.json( result )
+      }
+    })
+    .catch( error => {
+      console.log( chalk.red('There was an error in handleNewCouldDo: ')) // eslint-disable-line no-console
```

Comment:
Remove all these `// eslint-disable-line no-console` and add `"no-console": "off"` to your eslint rules